playbooks/slave_security.yml

```diff
@@ -1,19 +1,25 @@
+---
 - name: Secure Slave
-  hosts: job_nodes
+  hosts: "{{ target_hosts | default('job_nodes') }}"
   user: root
   vars:
     ip_address_whitelist: "{{ lookup('env', 'SSH_IP_ADDRESS_WHITELIST') }}"
-  pre_tasks:
-    - name: Install apt packages
+    ports_whitelist: "{{ lookup('env', 'FIREWALL_OPEN_PORTS') }}"
+  tasks:
+
```

Comment:
Nit: unnecessary newline

Reply:
It was intentionally added to make it a little easier to read. I find that when a bunch of text (ie the task and the vars above) have no whitespace between them it gets harder to read it clearly.